src/pages/index.js

```diff
@@ -1,6 +1,9 @@
 import "./index.css";
 import { enableValidation, validationConfig, resetValidation, disableButton } from "../scripts/validation.js";
-import { config } from "webpack";
+import Api from "../utils/Api.js";
+import { SetButtonText } from "../utils/helpers.js";
+
+
 
 const initialCards = [
```

Comment:
All constants (including `initialCards` and `validationConfig`) should be in `constants.js` in the `utils` folder to clean up the main file.


But you don't need it here anymore